# awesome-scRNA-analysis

### integration method
---
- #### scDREAMER
  - scDREAMER for atlas-level integration of single-cell datasets using deep generative model paired with adversarial classifier

### auto annotation
---
- #### CellSTAR
  - CellSTAR: a comprehensive resource for single-cell transcriptomic annotation

### visualization
---
- ##### Doheatmap
  - Scillus: https://github.com/xmc811/Scillus

### analysis pipeline
---
- ##### R
  - SingCellaR: Processing single-cell RNA-seq datasets using SingCellaR
  - Scillus: https://github.com/xmc811/Scillus

### identifying tumor cells
---
- #####
  - Ikarus: Identifying tumor cells at the singleâ€‘cell level using machine learning

### differential analysis
---
- ##### differentially expressed genes
  - muscat: muscat detects subpopulation-specific state transitions from multi-sample multi-condition single-cell transcriptomics data
- ##### differential abundance of celltype
  - Augur: Prioritization of cell types responsive to biological perturbations in single-cell data with Augur

### pseudotime analysis
---
- ##### estimating differentiation potency
  - CCAT: Ultra-fast scalable estimation of single-cell differentiation potency from scRNA-Seq data

- ###### pseudotime analysis
  - Cellrank: CellRank for directed single-cell fate mapping

### geneset analysis
---
- ##### Gene set activity
  - SCPA: Systematic single-cell pathway analysis to characterize early T cell activation
  - irGSEA: https://github.com/chuiqin/irGSEA
 
### cell cycle
---
- ##### cell cycle
  - tricycle: Universal prediction of cell-cycle position using transfer learning
 
### cell align across Atlas
---
- #### CellHint
  - Automatic cell-type harmonization and integration across Human Cell Atlas datasets
